docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBroker
+metadata:
+  name: example-ns-broker
+  namespace: ns-broker
+spec:
+  authInfo:
+    basic:
+      secretRef:
+        name: my-service-broker-auth
+        namespace: broker
+  url: http://my-service-broker.broker.svc.cluster.local
+```
+
+Once this resource is created, Service Catalog will query the Service Broker for the list of available Services and create corresponding `ServiceClass` and `ServicePlan` resources. These resources might look like this:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceClass
+metadata:
+  creationTimestamp: 2018-07-12T13:30:01Z
+  name: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+  namespace: ns-broker
+  resourceVersion: "13"
+  selfLink: /apis/servicecatalog.k8s.io/v1beta1/namespaces/ns-broker/serviceclasses/25434f16-d762-41c7-bbdd-8045d7f74ca6
+  uid: adfa2d9a-85d7-11e8-a4f3-2ae408f4a9e4
+spec:
+  bindable: true
+  bindingRetrievable: false
+  description: MySQL
+  externalID: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+  externalName: mysql-5-7
+  planUpdatable: false
+  serviceBrokerName: example-ns-broker
+status:
+  removedFromBrokerCatalog: false
+```
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServicePlan
+metadata:
+  creationTimestamp: 2018-07-12T13:30:02Z
+  name: 4c6932e8-30ec-4af9-83d2-6e27286dbab3
+  namespace: ns-broker
+  resourceVersion: "24"
+  selfLink: /apis/servicecatalog.k8s.io/v1beta1/namespaces/ns-broker/serviceplans/4c6932e8-30ec-4af9-83d2-6e27286dbab3
+  uid: ae8e23ac-85d7-11e8-a4f3-2ae408f4a9e4
+spec:
+  description: basic plan
+  externalID: 4c6932e8-30ec-4af9-83d2-6e27286dbab3
+serviceBrokerName: example-ns-broker
+  serviceClassRef:
+    name: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+status:
+  removedFromBrokerCatalog: false
+```
+
+The `ServiceInstance` resource has also been updated to allow you to use these resources just as you would the existing `ClusterServiceBroker`, `ClusterServiceClass` and `ClusterServicePlan` resources, except you will use them in the context of a namespace. For example, a `ServiceInstance` YAML that references a `ClusterServiceClass` and a `ClusterServicePlan` resource might look like this:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: example-mysql-instance
+  namespace: default
+spec:
+  clusterServiceClassExternalName: mysql-5-7
+  clusterServicePlanExternalName: basic
+```
+
+If you instead want to use the `ServiceClass` and `ServicePlan` namespace-scoped resources, the yaml might look like this:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: example-mysql-instance
+  namespace: default
+spec:
+  serviceClassExternalName: mysql-5-7
+  servicePlanExternalName: basic
+```
+
+In order to successfully create the resource using the namespace-scoped `ServiceClass` and `ServicePlan`, those resources must exist 
```

Comment:
I had to read this a few times to understand the rule. Instances can reference cluster-scoped classes/plans or classes/plans in the same namespace. They cannot reference classes/plans in another namespace.

Reply:
What @carolynvs just said - - '..or classes/plans in the **same namespace.**"  I think updating the wording to say this would help.